lib/project_types/extension/tasks/convert_server_config.rb

```diff
@@ -39,6 +39,7 @@ def call
               main: hash.dig("development", "entries", "main") || determine_default_entry_main(project_directory),
             )
           ),
+          can_access_network: hash.dig("can_access_network") || false,
```

Comment:
I would have personally preferred to do this:

```rb
can_access_network: hash.fetch("can_access_network") { false }
```

This is because `false` is a valid value for this property, so using `fetch` would save us from an unnecessary evaluation of the second clause of the OR statement. Micro-optimization, I know. But, I stuck with `dig` since it appears that we always use `hash.dig` in the CLI instead of `hash.fetch`.